sdc/datatypes/hpat_pandas_dataframe_functions.py

```diff
@@ -1867,13 +1867,93 @@ def _df_getitem_unicode_idx_impl(self, idx):
     ty_checker.raise_exc(idx, expected_types, 'idx')
 
 
+def df_getitem_single_label_loc_codegen(self, idx):
+    """
+    Example of generated implementation:
+        def _df_getitem_single_label_loc_impl(self, idx):
+            idx_list = []
+            for i in range(len(self._dataframe.index)):
+                if self._dataframe._index[i] == idx:
```

Comment:
What would happen if `_index` is `None`?

Also it is better to do it in parallel. Split it into chunks, create list per chunk and then merge them

Reply:
> What would happen if _index is None?
It is okay because I dont use dataframe._index in case of index = None